Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.ts

```diff
@@ -162,17 +167,17 @@ class BocAutoCompleteReferenceValue //TODO RM-7715 - Make the TypeScript classes
         {
           const row = document.createElement('li');
 
+          const imgContainer = document.createElement('div');
+          row.append(imgContainer);
```

Comment:
In this case, we create new elements that are not attached to the DOM yet (and never will because we only use it to generate HTML), so there are no updates to be made.

Even if they were attached to the DOM, shouldn't happen during JavaScript execution as the page is not rendered during execution. Thats why an infinite loop will freeze the page.